README.md

```diff
@@ -46,25 +35,16 @@ https://github.com/rmcolq/make_prg for a pipeline which can construct
 these PRGs from a set of aligned sequence files.
 
 Pandora can do the following for a single sample (read dataset):
-- Output inferred mosaic of reference sequences for loci (eg genes) from
-  the PanRG which are present
-- Output a VCF showing the variation found within these loci, with
-  respect to any reference path in the PRG.
-
-Soon, in a galaxy not so far away, it will allow:
-- discovery of new variation not in the PRG
+- Output inferred mosaic of reference sequences for loci (eg genes) from the PanRG which are present;
+- Output a VCF showing the variation found within these loci, with respect to any reference path in the PRG;
```

Comment:
Maybe we need to distinguish between a PanRG and a PRG as I am certain this will confuse some people. I assume in the second point we are referring to a local PRG?

Reply:
Yeah, that makes sense... these are not my changes, these are the conflicts I solved when doing the merge, but I did not read the README in details, I just solved the conflicts. I will add a commit after going through your points that will define what is a PRG (not defined here), and will fix all these PRG/PanRG mentions